add: issue templates (#127)
chdsbd/kodiak: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug report
about: Create a report to help us improve
title: ''
labels: bug
assignees: ''

---

> <!-- describe the problem you've encountered and the expected behavior. Screenshots of branch protection settings and Kodiak messages/behavior is always helpful -->

Reviewer: ```suggestion
<!-- 
- describe the problem you've encountered and the expected behavior
- screenshots of branch protection settings and Kodiak messages/behavior are helpful
-->
```
